src/intersection/intersection.spec.js

```diff
@@ -0,0 +1,43 @@
+import { arrayIntersection } from './intersection.js';
+
+describe('Intersect the elements that are equal and return them in a new array', () => {
+  describe('testing default parameters', () => {
+    it('Passing an object literal', () => {
```

Comment:
however 'it' part names have problems. 'It' part should describe what is the expected behavior of the code, not what is passed. For more clear examples : 

https://bignerdranch.com/blog/why-do-javascript-test-frameworks-use-describe-and-beforeeach/

and

https://mochajs.org/

An good example form last link:

You can read the code like poem

```Javascript
var assert = require('assert');
describe('Array', function() {
  describe('#indexOf()', function() {
    it('should return -1 when the value is not present', function() {
      assert.equal([1, 2, 3].indexOf(4), -1);
    });
  });
});

Reply:
@erkamguresen . oops, looks like I've a lot of example tests to rewrite in the module!